src/sdk/models/policies.ts

```diff
@@ -1,7 +1,8 @@
 import { IEntityIdentifier, IAuditedEntity } from './base';
 import { JobCategory } from './jobCategories';
 import { Job } from './jobs';
-import { TransferPath } from './transfers';
+import { TransferPath, TransferPlatformItem } from './transfers';
+import { AuditCategory } from './auditCategories';
```

Comment:
We should refactor [this resource](https://github.com/portalarchitects/skysync-cli/blob/master/src/sdk/resources/auditCategories.ts#L5) since that is transfer specific and there will be policy-based audit categories coming [here](https://github.com/portalarchitects/skysync-vnext/pull/2458/files#diff-460d13e2b0c313d2ea00a49ec8e52b0d).